### PR TITLE
[Merged by Bors] - feat: updatd connector documentation

### PR DESCRIPTION
**Pull request by EstebanBorai**
Updates documentation to provide example on usage

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,39 +8,115 @@ HTTP sink connector reads records from data streaming and generates an HTTP requ
...
 > Supports HTTP/1.0, HTTP/1.1, HTTP/2.0 protocols.
 
+### Configuration
 
-See [docs][1] here.
+HTTP Sink is configured using a YAML file:
 
-### Configuration
+```yaml
+# config-example.yaml
```
Comment (morenol) [inline README.md:16]:
```suggestion
# config-example.yaml
apiVersion: 0.1.0
```

File: README.md [hunk 2]
```diff
@@ -8,39 +8,115 @@ HTTP sink connector reads records from data streaming and generates an HTTP requ
...
+```
 
-This is an example of simple connector config file:
+Then configure the HTTP Request to be sent using a YAML file following the
+connector configuration schema. The following configuration will send a POST
+HTTP request to `http://httpbin.org/post`.
 
 ```yaml
-# config-example.yaml
+# config.yaml
```
Comment (morenol) [inline README.md:52]:
```suggestion
# config.yaml
apiVersion: 0.1.0
```

File: README.md [hunk 3]
```diff
@@ -8,39 +8,117 @@ HTTP sink connector reads records from data streaming and generates an HTTP requ
...
+meta:
+  version: 0.1.0
+  name: my-http-sink
+  type: http-sink
+  topic: http-sink-topic
+http:
+  endpoint: "http://127.0.0.1/post"
+  headers:
+    - "Authorization: token MySecretToken"
+    - "Cache-Control: no-cache"
```
Comment (morenol) [inline README.md:27]:
```suggestion
  topic: http-sink-topic
http:
  endpoint: "http://127.0.0.1/post"
  secrets:
    - name: HTTP_TOKEN
  headers:
    - "Authorization: token ${{ secrets.HTTP_TOKEN }}"
    - "Cache-Control: no-cache"
```

File: README.md [hunk 4]
```diff
@@ -8,39 +8,119 @@ HTTP sink connector reads records from data streaming and generates an HTTP requ
...
+> You can see active connectors by running the following command:
+>
+> ```bash
+> fluvio cloud connector list
+> ```
+
+Check connector logs by running
+
+```bash
+fluvio cloud connector logs <NAME>
```
Comment (ajhunyady) [inline README.md:83]: I know this is a closed `pr`, but we should change <NAME> to `fluvio cloud connector logs httpbin` as the readme focuses on this particular use case.

Comment (EstebanBorai) [reply README.md:83]: Of course!

**Pull request closed without merging**